### ISSUE TITLE
Address validation fails, but Google perfectly finds the address

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/peeringdb#214]
https://peeringdb.com/fac/1778

**Comment by arnoldnipper** [comment]
https://www.peeringdb.com/fac/569 ... is address validation working at all?

**Comment by vegu** [comment]
It should definitely find those, since we use geocoding as a form to provide coordinates as well as validate addresses it might be that the accuracy filtering is currently set to strictly. There are plenty of facilities that got geocoded correctly, however further tweaking is required.

**Comment by vegu** [comment]
Looks like the "subpremise" and "political" location types are currently not counted towards a validation when they should.

**Comment by arnoldnipper** [comment]
What do you mean by "subpremise"? The Adress2 field? This is taken into account. And that is the reason why 569 fails

**Comment by vegu** [comment]
Ok so for 1778 there are currently two reasons why it fails validation:

1. The zipcode we have is 76240 when it should be 76246, also the city name should just be "Querétaro" or "Santiago de Querétaro"
2. Google API returns the location type "establishment" which is used for as so far uncategorized locations on their end, we are currently ignoring that location type (same as with subpremise), although i think it's probably reasonable to count it as a validation match

**Comment by vegu** [comment]
I think this can be moved to review/backlog - until it is fixed it is possible for admins to manually validate/geocode by setting lat and lon in the facility view inside the admin ui

**Comment by mahtin** [comment]
@vegu - thanks for this; however, I wonder if there's a simpler version of the address that would validate? Is this address (which works in Google Maps) too much? 
```
    Privada de la Princesa 4,
    Parque Industrial el Marques, 76246
    Santiago de Querétaro, Qro.,
    Mexico
```
A Google [search](https://www.google.com/search?q=Privada+de+la+Princesa+4,+Parque+Industrial+el+Marques,+Quer%C3%A9taro,+MX) for:
```
    Privada de la Princesa 4,
    Parque Industrial el Marques,
    Querétaro, MX
```
But that's google vs. PeeringDB. What advice should the admin be given?

**Comment by vegu** [comment]
Hm, so for 1778 any variation of the address i've tried comes back typed as "establishment" - we'd want a street_address or premise type, to be honest probably easiest to just have a pdb admin set the coordinates manually in the control panel in such instances. 

569 might be possible if Address 2 is blanked, i have not tried yet. However i want to note that now that the problem is identified that this is a super quick fix on the code side of things, so with proper prioritization should be fixed sooner rather than later.

**Comment by vegu** [comment]
I went ahead and manually validated the address for 1778

**Comment by arnoldnipper** [comment]
569 works if Address 2 is blanked. I tried it already earlier and should have mentioned it. Sorry!

**Comment by vegu** [comment]
It's a simple fix and i essentially fixed this while i was investigating to reproduce, so moving this to the Done column as it will be tagged in the next release.

fixed in b5c24efe

**Comment by vegu** [comment]
This changes is live on beta, and ~500 addresses that could not be found before are now geocoded correctly.
